app/src/app/components/stix/stixid-property/stixid-property.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="stix-header text-deemphasis">
+    <div class="content" [ngClass]="{'icon-button': config.object['stixID']}">
```

Comment:
In line with my comment above, we should be able to safely assume there is a STIX ID at this point, and so the `icon-button` style class can be added directly without using `ngClass`. This would help simplify the template and improve readability.

Minor change, but I'd also recommend reconsidering the use of the "content" class name here. This style class is typically associated with elements inside of a custom labeled box (e.g. the box around the ATT&CK ID), which isn't the intended use in this context. Since it is used for padding, I'd recommend using only the `.icon-button` class to apply it, or create a more context-specific class name.